tests/unit/matrix/test_cubemeasure.py

```diff
@@ -1316,10 +1345,22 @@ def it_knows_its_weighted_counts(self, raw_weighted_counts):
             )
         )
 
+    def it_knows_its_zscores(self, raw_weighted_counts):
+        weighted_cube_counts = _CatXMrWeightedCubeCounts(None, raw_weighted_counts)
+        assert weighted_cube_counts.zscores.tolist() == [
+            [-1.7549928774784245, -1.6818357317441646, -1.754992877478425],
+            [1.7549928774784245, 1.6818357317441637, 1.754992877478425],
+        ]
```

Comment:
This will not pass reliably because it tests floats for equality. Use `pytest.approx()` on the left side and then you can drop the `.tolist()` as well. The `.tolist()` pattern only applies for integer results.